components/Common/Sidebar/SidebarGroup/index.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import type { FC } from 'react';
+
+import SidebarItem from '@/components/Common/Sidebar/SidebarItem';
+
+import styles from './index.module.css';
+import type { ActiveItem, SidebarGroupType } from '..';
+
+type SideBarGroupProps = SidebarGroupType & {
+  activeItem?: ActiveItem;
+  setActiveItem: (item: ActiveItem) => void;
+};
+
+const SidebarGroup: FC<SideBarGroupProps> = ({
+  groupName,
+  items,
+  activeItem,
+  setActiveItem,
+}) => {
+  const handleSideBarItemClick = (title: string) => {
+    setActiveItem({ groupName, title });
+  };
+
+  return (
```

Comment:
```suggestion
}) => (
```

No need for the click handler and return statement